metadata/schema/resources/user.json

```diff
@@ -7,14 +7,14 @@
   "properties": {
     "user_id": {
       "description": "Native ID of the user (e.g. arXiv user ID).",
-      "type": "string"
+      "type": "integer"
     },
-    "first_name": {
+    "forename": {
       "description": "Forename of the user",
       "type": "string"
     },
-    "last_name": {
-      "description": "Surnamne of the user",
+    "surname": {
```

Comment:
I haven't looked to understand exactly how this schema is being used, so this comment may not be relevant: there could be some user properties to add here, including (but not limited) to `country`, `career_status` and `url`.

Reply:
Yeah, I think we should come around to user metadata as its own issue -- lots of things to think about.